app/packs/stylesheets/decidim/_decidim-settings.scss

```diff
@@ -5,3 +5,4 @@
 @include foundation-button;
 @include foundation-card;
 @include foundation-tabs;
+// @include foundation-forms;
```

Comment:
This line is commented. To keep the code cleaner, we should delete it unless it has been added for one reason I don't know